motionplan/planManager.go

```diff
@@ -569,6 +573,7 @@ func (pm *planManager) plannerSetupFromMoveRequest(
 		opt.ScoreFunc = tpspace.PTGSegmentMetric
 
 		planAlg = "tpspace"
+		opt.relativeInputs = true
```

Comment:
Is this fixing a bug?

Reply:
No its a new addition.  As I wrote in planner_options this is the way the algorithm specifies if its plans are returned with relative inputs